wordle-web/src/components/AppBar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <v-card class="mx-auto" max-width="448" zIndex="0">
+    <v-layout>
+      <v-app-bar density="compact">
+        <template v-slot:prepend>
+          <v-btn icon="mdi-menu" @click.stop="hamburgerDrawer = !hamburgerDrawer"> </v-btn>
+        </template>
+        <v-icon icon="mdi-head-cog" @click.stop="$router.push('/')" size="x-large"></v-icon>
+        <v-app-bar-title @click.stop="$router.push('/')">Wordle</v-app-bar-title>
+        <template v-slot:append>
+          <v-btn icon="mdi-cog" @click.stop="settingDrawer = !settingDrawer"> </v-btn>
+        </template>
+      </v-app-bar>
+
+      <v-navigation-drawer temporary v-model="hamburgerDrawer" location="left">
+      </v-navigation-drawer>
+
+      <v-navigation-drawer v-model="settingDrawer" location="right">
+        <v-list>
+          <v-btn @click="setInverseTheme"> Inverse Theme </v-btn>
+          <v-btn @click="setDarkTheme"> Dark Theme </v-btn>
+        </v-list>
+      </v-navigation-drawer>
+      <v-main> </v-main>
```

Comment:
This didn't get fixed, but after our various other changes it looks like this line is what keeps the main game content from ignoring the AppBar and aligning itself with the top of the screen instead of the bottom of the AppBar. We could investigate it a bit more but it feels a little beyond the scope of what we know so far.